x-pack/plugins/index_lifecycle_management/public/extend_index_management/components/index_lifecycle_summary.tsx

```diff
@@ -140,7 +140,9 @@ export class IndexLifecycleSummary extends Component<Props, State> {
                 id="xpack.indexLifecycleMgmt.indexLifecycleMgmtSummary.phaseDefinitionTitle"
               />
             </EuiPopoverTitle>
-            <EuiCodeBlock lang="json">{JSON.stringify(ilm.phase_execution, null, 2)}</EuiCodeBlock>
+            <EuiCodeBlock language="json">
```

Comment:
Thanks for changing this prop! Could you please include a note in the PR description that mention this change? For example: "Updated ILM UI code blocks to use correct `language` prop instead of `lang`."